core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -109,7 +109,7 @@
 
   <ul>
     <li [ngClass]="{'active': currentUrl === '/learn/math'}">
-      <a (click)="toggleClassroomSubmenu()" class="protractor-mobile-test-classroom">
+      <a (click)="toggleClassroomSubmenu()" class="protractor-mobile-test-classroom oppia-sidebar-expandable-list">
```

Comment:
A bit confused, what does this class do?

Reply:
In `.ts` file i'm selecting this element by queryselector method for stopping the propagation of click event on this element.